templates/masters-conference/index.html

```diff
@@ -105,8 +105,11 @@ <h2>Scaling and delivering applications with high performance and low friction</
       <p>How do you support a diverse infrastructure that spans six data centres, three continents and the public cloud? Adobe uses open-source technologies, including Ubuntu, Kubernetes and OpenStack, to craft a feature-rich platform that developers can build on to best serve their customers. </p>
 
       <p>
-        <a href="/masters-conference/contact-us" class="p-button--neutral is-inline js-invoke-modal">Get notified about the full keynote presentation</a>
-        <br />&mdash;&nbsp;coming in January
+        <a href="https://www.youtube.com/watch?v=IUgODRm9Soc" class="p-button--neutral"><span class="p-link--external">See the full keynote presentation</span></a>
+      </p>
+
+      <p>
+        <a href="/masters-conference/joe-sandoval">Read more about Extended BPF&nbsp;&rsaquo;</a>
```

Comment:
Is the new page still about Extended BPF?

Reply:
My bad, fixed